mk/pulp.mk

```
@@ -0,0 +1,18 @@
+CS_COMPOSE_FILE ?= "deployments/docker-compose.yml"
+PULP_COMPOSE_FILES ?= "compose_files/pulp/pulp-oci-images/images/compose/docker-compose.yml"
+PULP_COMPOSE_OPTIONS=PULP_POSTGRES_PATH="pulp_db" PULP_STORAGE_PATH="pulp_storage"
+
+ifeq ($(DEPLOY_PULP),true)
+	PULP_COMPOSE_COMMAND=$(PULP_COMPOSE_OPTIONS) $(DOCKER)-compose  -f $(PULP_COMPOSE_FILES) up --detach
+	PULP_COMPOSE_DOWN_COMMAND=$(PULP_COMPOSE_OPTIONS) $(DOCKER)-compose  -f $(PULP_COMPOSE_FILES) down
+
+else
+	PULP_COMPOSE_COMMAND=echo "Skipping pulp deploy, set DEPLOY_PULP=true to deploy"
+	PULP_COMPOSE_DOWN_COMMAND=$(PULP_COMPOSE_COMMAND)
+endif
```

I suggest two changes here:
- Remove indent as they are not rules.
- Set `PULP_COMPOSE_DOWN_COMMAND=true` when `DEPLOY_PULD` is not `true`
```suggestion
ifeq ($(DEPLOY_PULP),true)
PULP_COMPOSE_COMMAND=$(PULP_COMPOSE_OPTIONS) $(DOCKER)-compose  -f $(PULP_COMPOSE_FILES) up --detach
PULP_COMPOSE_DOWN_COMMAND=$(PULP_COMPOSE_OPTIONS) $(DOCKER)-compose  -f $(PULP_COMPOSE_FILES) down
else
PULP_COMPOSE_COMMAND=echo "Skipping pulp deploy, set DEPLOY_PULP=true to deploy"
PULP_COMPOSE_DOWN_COMMAND=true
endif
```

---

I actually with -p cs  

using pulp, caused names of 'pulp_pulp_api', which seemed weird, and everything seemed to work with just 'cs'